notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings.ipynb

```diff
@@ -470,13 +418,13 @@
         "id": "1124422cc200"
       },
       "source": [
-        "#### Instantiate the text encoding model\n",
+        "### Instantiate the text encoding model\n",
         "\n",
         "Use the [sentence-t5 encoder](https://tfhub.dev/google/sentence-t5/st5-base/1) developed by Google for converting text to embeddings.\n",
         "\n",
         "> The sentence-T5 family of models encode text into high-dimensional vectors that can be used for text classification, semantic similarity, clustering and other natural language processing tasks.\n",
         ">\n",
-        "> Our model is built on top of T5 (i.e. the Text-To-Text Transfer Transformer). It is trained on a variety of data sources and initialized from pre-trained T5 models with different model sizes. The input is variable-length English text and the output is a 768-dimensional vector. The sentence-T5 base model employs a 12-layer transformer architecture as the T5 base model does."
+        "> Our model is built on top of T5 (i.e. the Text-To-Text Transfer Transformer). It's trained on a variety of data sources and initialized from pre-trained T5 models with different model sizes. The input is variable-length English text and the output is a 768-dimensional vector. The sentence-T5 base model employs a 12-layer transformer architecture as the T5 base model does."
```

Comment:
Don't use "Our". Suggest changing to "The model". Also, should be i.e.,:
- "The model is built on top of T5 (i.e., the Text-To-Text Transfer Transformer). "

Redo do the last sentence:
- "The sentence-T5 base model employs a 12-layer transformer architecture as does the T5 base model."